exporters/stdout/metric.go

```diff
@@ -44,13 +45,37 @@ type line struct {
 
 	Quantiles []quantile `json:"Quantiles,omitempty"`
 
+	Histogram histogram `json:"histogram,omitempty"`
```

Comment:
Can we follow the other naming structure here and make this capital.

```suggestion
	Histogram histogram `json:"Histogram,omitempty"`
```